sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java

```diff
@@ -468,6 +468,16 @@ public int update(@NonNull String table, @NonNull ContentValues values,
      */
     @WorkerThread
     boolean yieldIfContendedSafely(long sleepAmount, TimeUnit sleepUnit);
+
+    /**
+     * Closes the object and release any system resources it holds.
```

Comment:
In the test we reference `Closeable` directly so I think we still need the throw there